spaced-repetition-back-end/knexfileOLD.js

```diff
@@ -0,0 +1,32 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/mealprepdb.sqlite3'
```

Comment:
This knex development object is going to need to be changed. client needs to be pg, and the connection will need to be changed as well, like this:
```
   client: 'pg',
    connection: {
      "user": "development",
      "password": "development",
      "database": "srsly",
      "host": "127.0.0.1",
      "port": 5432
    },
    pool: {
      min: 2,
      max: 10
    },
```
Also found a nice reference guide for local Postgres setup that will help get db set up and do user privileges:
https://www.codementor.io/engineerapart/getting-started-with-postgresql-on-mac-osx-are8jcopb

Reply:
thats the old knex file i was using for reference, i can delete it now